### PR TITLE
Change CI to removed client code generation in knora-api

### LINKED ISSUE #155: Adapt to removed client code generation in knora-api

### PR DESCRIPTION
**Pull request by subotic**
resolves #155

### CODE REVIEW

**Comment by subotic** [comment]
@tobiasschweizer The test data has changed and so the integration of test data fails (missing page.json)

**Comment by tobiasschweizer** [comment]
I will have a look at that tomorrow. I think all non anything test data files have been replaced.

**Comment by tobiasschweizer** [comment]
@benjamingeer @mrivoal I adapted the integration of admin API test data. I noticed that 

`test/data/api/admin/projects/get-project-keywords-response.json` contains 

```
{"keywords":[]}
```

I think the array should contain some keywords for a realistic test scenario.

**Comment by benjamingeer** [comment]
> I think the array should contain some keywords for a realistic test scenario.

I thought I had done that. Is your `knora-test` repository up to date with the latest `anything-data.ttl`?

**Comment by tobiasschweizer** [comment]
I have run ` make init-db-test` on `develop` and then `make stack-restart-api`.

File: src/api/admin/projects/projects-endpoint.spec.ts [hunk 1]
```diff
@@ -134,8 +134,9 @@ describe("ProjectsEndpoint", () => {
             knoraApiConnection.admin.projectsEndpoint.getProjectKeywords("http://rdfh.ch/projects/00FF").subscribe(
                 (response: ApiResponseData<KeywordsResponse>) => {
 
-                    expect(response.body.keywords.length).toEqual(14);
-                    expect(response.body.keywords[0]).toEqual("Basel");
+                    // TODO: fix JSON test data
```
Comment (tobiasschweizer) [inline src/api/admin/projects/projects-endpoint.spec.ts]: this test should be adapted once the test data is correct

Comment (tobiasschweizer) [reply src/api/admin/projects/projects-endpoint.spec.ts]: @benjamingeer Now I got 2 keywords on github-ci, maybe my local repo was not up to data. I will check this.

**Comment by tobiasschweizer** [comment]
> > I think the array should contain some keywords for a realistic test scenario.
> 
> I thought I had done that. Is your `knora-test` repository up to date with the latest `anything-data.ttl`?

Sorry, you are right. It was a local issue.

**Comment by tobiasschweizer** [comment]
@subotic I think I fixed it.

**Comment by subotic** [comment]
Great, thanks :-)

**Pull request merged**